src/main/java/nextstep/fp/StreamStudy.java

```diff
@@ -28,6 +29,13 @@ public static void printLongestWordTop100() throws IOException {
         List<String> words = Arrays.asList(contents.split("[\\P{L}]+"));
 
         // TODO 이 부분에 구현한다.
+        words.stream()
+                .filter(w -> w.length() > 12)
```

Comment:
```suggestion
                .filter(it -> it.length() > 12)
```

저는 그냥 스트림일때는 `it`,`value`, `target` 이런 일반화된 단어를 사용하긴해요 🤔 

지금이 잘못되었다는것은 아닙니다!